kubeflow/katib/studyjobcontroller.libsonnet

```diff
@@ -261,7 +290,20 @@
           namespace: namespace,
         },
         data: {
-          "defaultWorkerTemplate.yaml": "apiVersion: batch/v1\nkind: Job\nmetadata:\n  name: {{.WorkerId}}\n  spec:\n  template:\n    spec:\n      containers:\n      - name: {{.WorkerId}}\n        image: alpine\n      restartPolicy: Never",
+          "defaultWorkerTemplate.yaml": |||
+            apiVersion: batch/v1
+            namespace: %(ns)s
+            kind: Job
+            metadata:
+              name: {{.WorkerId}}
+            spec:
+              template:
+                spec:
+                  containers:
+                  - name: {{.WorkerId}}
+                    image: alpine
+                  restartPolicy: Never
+          |||% {ns: namespace},
```

Comment:
Is this a typo? Why is "% {ns:namespace} outside the verbatim string?

Reply:
This is not typo. It is string formatting same as example 3 in https://jsonnet.org/ .